ui/app/components/namespace-picker.ts

```diff
@@ -139,6 +139,27 @@ export default class NamespacePicker extends Component {
     return this.namespaceOptions.slice(0, this.batchSize);
   }
 
+  @action
+  adjustElementWidth(element: HTMLElement): void {
+    // Hide the element so that it doesn't affect the layout
+    element.style.display = 'none';
+
+    let maxWidth = 240; // Default minimum width
+    const namespaceLinks = document.querySelectorAll('[data-test-namespace-link]');
+    namespaceLinks.forEach((checkmark: Element) => {
+      const width = (checkmark as HTMLElement).offsetWidth;
+      if (width > maxWidth) {
```

Comment:
Does this wrap at any point or will it always expand to the longest namespace? Wondering if we should have/need some sort of max limit 🤔

Reply:
The HDS component has a maxWidth of 400px, so if it's > 400px it will wrap. 

For example, in this case the help text (_"Enter a full path..."_) at the top is the element w/ the adjusted width, and you can see that it is wrapped to 2 lines:
![Screenshot 2025-05-12 at 10 31 55 AM](https://github.com/user-attachments/assets/48eddaae-917e-499d-bc94-574740fc8b0d)

Reply:
Awesome! Thanks for confirming